support/fixtures/factories/contxtUserEvent.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('contxtUserEvent')
+  .option('fromServer', false)
+  .attrs({
+    eventId: () => factory.build('event').id,
+    createdAt: () => faker.date.past().toISOString(),
+    id: () => faker.random.uuid(),
```

Comment:
These are not UUIDs -- you could use the contxtUser to build the right type

Reply:
I'm wrong. I misread. Apologies.

Reply:
Chatted with Greg, we agree that it should be a uuid instead.